### ISSUE TITLE
Some confusion about using the ground-truth mask file of DTU.

### ISSUE THREAD

**Issue by prstrive** [xxlong0/SparseNeuS#20]
In the training stage, I noticed that you adopted the ground-truth mask file provied by DTU to sample more rays withine valid regions. However, these ground-truth mask files can mask out most non-object regions. And this seems to introduce the Mask prior like IDR, or is there something I'm misunderstanding

**Comment by flamehaze1115** [comment]
Hello. This masks in the dataloader are not the ground truth masks provided by IDR. 
IDR generates the masks by human annotation, in the DTU dataset, all the background parts like the table floors are masked out.
We just notice that the images of DTU have black backrgound regions (rgb=0), and these regions contribute very small errors in the total loss. Our masks just indicate the regions where rgb values are 0.
We adopt such a simple dataset prior to sample more rays in the regions with nonzero rgb values, which can help the training be faster.  Training without such masks is OK and just needs a bit more epochs.
